core/src/datayoga_core/resources/schemas/connections.schema.json

```diff
@@ -21,12 +21,18 @@
               "minimum": 1,
               "maximum": 65535
             },
+            "driver": { "description": "Driver", "type": "string" },
             "database": { "description": "DB name", "type": "string" },
             "user": { "description": "DB user", "type": "string" },
             "password": { "description": "DB password", "type": "string" },
             "connect_args": {
               "description": "Additional arguments to use when connecting to the DB",
               "type": "object"
+            },
+            "debug": {
+              "description": "Debug mode",
```

Comment:
add comment explaining this will echo all SQL commands to stdout

Reply:
done